lib/solidus_dev_support/rspec/feature_helper.rb

```diff
@@ -12,6 +12,10 @@
 require 'capybara-screenshot/rspec'
 require 'solidus_dev_support/rspec/capybara'
 
+Capybara::Screenshot.register_driver(:solidus_chrome_headless) do |driver, path|
+  driver.browser.save_screenshot(path)
+end
+
```

Comment:
Thoughts on moving it to https://github.com/solidusio/solidus_dev_support/blob/8d1d6dace36d3d274b94224d1d516c5adbc7227b/lib/solidus_dev_support/rspec/capybara.rb ?

Reply:
Then `cabybara.rb` needs to be sure to be run before we `require "capybara-screenshot"` in here or we move the `require` to that file as well.

Reply:
I think it would make sense to have anything related to capybara inside capybara.rb

Reply:
I moved the require into capybara.rb so it is all in one place